package.yaml

```diff
@@ -75,10 +82,18 @@ library:
     - errors
     - http-conduit
     - memory
+    - network-uri
     - pem
+    - regex-tdfa
     - text
     - x509
     - x509-validation
+  when:
+    - condition: 'flag(development)'
+      then:
+        ghc-options: '-DDEVELOPMENT'
+      else:
+        ghc-options: ''
```

Comment:
```suggestion
      then:
        ghc-options:
          - -DDEVELOPMENT
```

Reply:
Unfortunately you need an `else`.

Reply:
Oh sorry. That's only because you used `then`. https://github.com/sol/hpack#conditionals. Let me edit my suggestion.